services/task.service.js

```diff
@@ -51,3 +51,30 @@ export const updateTask = async (id, {title, priority, tags, expired_date}) => {
     throw new Error(INTERNAL_SERVER_ERROR.message);
   }
 };
+
+export const updateTaskStatus = async (id) => {
+  try {
+    const updatedTask = await Task.updateOne(
+      {
+        _id: id
+      },
+      {
+        status: 2
```

Comment:
Status should be passed from the client. It is `updateTask` not `completeTask` api. What if you accidentaly complete it?

Reply:
Yes, but I think that it must be impossible on client to change a status from completed to new. If tasks is completed, that`s finished. And it is why we ask user about this in confirm dialog. "Are you sure that you want to complete this task?". Because after this you can not change status anymore. No?

Reply:
Discussed.